client/my-sites/plugins/jetpack-plugins-setup/index.jsx

```diff
@@ -473,10 +476,13 @@ const PlansSetup = React.createClass( {
 	},
 
 	render() {
-		const { translate } = this.props;
+		const {
+			sitesInitialized,
+			translate
+		} = this.props;
```

Comment:
Yeah, I agree. I left it like that though, because we'd have to alias the import for that, and I don't think it's something we should do in this PR.